test/preflight/fly_deploy_test.go

```diff
@@ -323,3 +323,14 @@ func TestFlyDeploy_CreateBuilderWDeployToken(t *testing.T) {
 	f.OverrideAuthAccessToken(f.Fly("tokens deploy").StdOutString())
 	f.Fly("deploy")
 }
+
+func TestDeployManifest(t *testing.T) {
+	f := testlib.NewTestEnvFromEnv(t)
+
+	appName := f.CreateRandomAppName()
+	f.Fly("launch --org %s --name %s --region %s --image nginx --internal-port 80 --ha=false", f.OrgSlug(), appName, f.PrimaryRegion())
+
+	f.Fly("deploy --export-manifest manifest.json")
```

Comment:
maybe we should check that the manifest contains the values we'd expect. We could either do this here, or in a manifest_test.go unit test. That way as we add new things to the manifest, we can ensure that exporting it is adding everything we expect